source/administration/important-upgrade-notes.rst

```diff
@@ -5,10 +5,10 @@ Important Upgrade Notes
    Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ has come to the end of its life cycle as of April 15, 2021. Upgrading to Mattermost Server v5.31 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ or later is required.
 
 .. important::
-   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
+   Due to the introduction of new features (Shared Channels and Folded Reply Threads) the performance of the migration process for the 5.35 (May 16, 2021) release has been noticeably affected. Depending on the size, type and version of the database, longer than usual upgrade times should be expected. These can vary from a couple of minutes (average case) to hours (worst case, MySQL 5.x only). A moderate to significant spike in database CPU usage should also be expected during this process. `The queries to run ahead of time to minimize the impact are provided here <https://gist.github.com/streamer45/9aee4906639a49ebde68b2f3c0f924c1>`_. 
```

Comment:
Maybe I'd keep the last line more generic as the document doesn't really list queries but offers some more details on the performance impact of the migration and suggests possible mitigation strategies.